lib/elixir/lib/io/ansi.ex

```diff
@@ -21,6 +21,20 @@ defmodule IO.ANSI do
   [ANSI escape sequences](https://en.wikipedia.org/wiki/ANSI_escape_code)
   are characters embedded in text used to control formatting, color, and
   other output options on video text terminals.
+
+  ## Examples
+
+  Because the ANSI escape sequences are embedded in text, the normal usage of
+  these functions is to concatenate their output with text.
+
+      formatted_text = IO.ANSI.blue_background() <> "Example" <> IO.ANSI.reset()
+      IO.puts formatted_text
+
+  A higher level and more convenient API is also available via `IO.ANSI.format/1`,
+  where you use atoms to represent each ANSI escape sequence:
+
+      IO.puts IO.ANSI.format [:blue_background, "Example", :reset]
```

Comment:
Nitpick: format code as the formatter would.

```suggestion
      IO.puts(IO.ANSI.format([:blue_background, "Example", :reset]))
```

or 

```elixir
formatted_text = IO.ANSI.format([:blue_background, "Example", :reset])
IO.puts(formatted_text)
```